src/pages/TabDetails.tsx

```diff
@@ -186,40 +222,102 @@ const TabDetails = () => {
   })}`;
 
   const p_style = {
-    fontSize: "10px" as const,
-    color: "var(--ion-color-basic)" as const,
-    textAlign: "center" as const,
+    fontSize: "13px" as const,
+    color: "var(--ion-color-light)" as const,
+    textAlign: "left" as const,
+    marginBottom: "5px" as const,
   };
 
   const h_style = {
-    fontWeight: "bold" as const,
-    color: "var(--ion-color-dark-basic)" as const,
-    textAlign: "center" as const,
+    fontSize: "20px" as const,
+    color: "var(--ion-color-light)" as const,
+    textAlign: "left" as const,
+  };
+
+  const History = () => {
```

Comment:
I didn't see any nested JSX component except of HOC using, maybe it's better to move to upper level out of `TabDetails` component?

Reply:
Fixed